sechub-wrapper-xray/src/test/java/com/mercedesbenz/sechub/wrapper/xray/report/XrayWrapperReportVulnerabilityMapperTest.java

```diff
@@ -49,7 +49,7 @@ void mapVulnerabilities_throws_illegalArgumentException() {
     }
 
     @Test
-    void mapVulnerabilities_throws_() {
+    void mapVulnerabilities_throws_xrayWrapperReportException() {
         /* prepare */
         HashMap<String, CycloneDXVulnerabilityHelper> vulnerabilityHashMap = new HashMap<>();
```

Comment:
```suggestion
        Map<String, CycloneDXVulnerabilityHelper> vulnerabilityHashMap = new HashMap<>();
```
It is always a good idea to use interfaces on the left side and hide the details - it reduces time for refactoring, changes and also hides implementation details.